File: src/Iniciante/1041.py
a,b = input().split(" ")
a = float(a)
b = float(b)

if a > 0 and b > 0:
    print(f"Q1")

elif a < 0 and b >0:
    print(f"Q2")

elif a < 0 and b < 0:
    print(f"Q3")

elif a > 0 and b < 0:
    print(f"Q4")

elif a == 0 and b == 0:
    print(f"Origem")

elif a == 0:
    print(f"Eixo Y")

elif b == 0:
    print(f"Eixo X")

File: src/Iniciante/1041.java
package Iniciante;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        float c1 = input.nextFloat();
        float c2 = input.nextFloat();
        if (c1 > 0 && c2 > 0) {
            System.out.println("Q1");
        }
        if (c1 < 0 && c2 > 0) {
            System.out.println("Q2");
        }
        if (c1 < 0 && c2 < 0) {
            System.out.println("Q3");
        }
        if (c1 > 0 && c2 < 0) {
            System.out.println("Q4");
        }
        if (c1 == 0 && c2 == 0) {
            System.out.println("Origem");
        }
        if ((c1 == 0 && c2 != 0)) {
            System.out.println("Eixo Y");
        }
        if ((c1 != 0 && c2 == 0)) {
            System.out.println("Eixo X");
        }

    }

}